src/__experimental__/components/checkable-input/checkable-input.component.js

```diff
@@ -59,8 +67,12 @@ CheckableInput.propTypes = {
   children: PropTypes.node,
   /** Toggles disabling of input */
   disabled: PropTypes.bool,
-  /** Toggles error styles */
-  error: PropTypes.bool,
+  /** Status of error validations */
```

Comment:
I think all of these props should be more descriptive. What's the difference between passing a string and a bool?